src/SWP/Bundle/CoreBundle/Tests/Controller/RegistrationController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Tests\Controller;
+
+use SWP\Bundle\FixturesBundle\WebTestCase;
+use Symfony\Component\Routing\RouterInterface;
+
+class RegistrationController extends WebTestCase
+{
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp()
+    {
+        self::bootKernel();
+        $this->initDatabase();
+        $this->loadCustomFixtures(['tenant']);
+        $this->router = $this->getContainer()->get('router');
+    }
+
+    public function testRevisionPublishing()
+    {
+        $client = static::createClient();
+        $client->enableProfiler();
```

Comment:
nope, as it allows us to get details of send email. Later we use code from profiler in assertions.